builtin/game/item_entity.lua

```diff
@@ -36,6 +36,9 @@ core.register_entity(":__builtin:item", {
 	itemstring = '',
 	physical_state = true,
 	age = 0,
+	-- The items are mergeable as long as this property is not false,
+	-- so we use nil to save a few bytes unless it will be set.
+	mergeable = nil,
```

Comment:
not sure if this is a good idea if you don't explicitly check for false but do a "if (mergable)" this will result in exactly opposit you want.
Yes I know this is commented but it's still gonna cause issues for sure.